documentation/gapc on windows.txt

```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore

```

Comment:
typo: St_r_ore

Reply:
is it possible to give a URL for that? Which Ubuntu version is currently used? 18.04?

Reply:
I made sure which Ubuntu version to use but the "Windows Store" is an app and I dont know if its possible to link a spezific source.  I think it should be clear what to do. Its like using Appstore or Playstore on your phone.